README.md

```diff
@@ -56,7 +56,7 @@ Once you setup the plugin correctly, you can call the `:generateSwagger` gradle
 
 In order to run this gradle plugin you need to fulfill the following requirements:
 
-* Gradle 5.x - This plugin uses Gradle 5 features, and you will need to setup your Gradle wrapper to use 5.4.1 or more.
+* Gradle 6.x - This plugin uses Gradle 6 features, and you will need to setup your Gradle wrapper to use 6.1.1 or newer.
```

Comment:
Please amend this change. Though that is true for the source code on `master`, the latest released version (`v1.4.1`) works fine on Gradle 5.4.1.

If we merge this change, users will be confused by this discrepancy between the README and the latest stable. We can update it once we release `v2.x`